packages/amplication-client/src/VersionControl/PendingChange.scss

```diff
@@ -23,6 +29,16 @@
     margin-left: var(--double-spacing);
     margin-right: var(--default-spacing);
 
+    &::before {
+      content: "";
+      position: absolute;
+      display: block;
+      left: 10px;
+      border-right: 1px solid var(--black30);
+      top: -6px;
+      bottom: -6px;
```

Comment:
if you can use existing variables please do so. 
if you need exactly these values, please create SCSS parameters at the top of the file